musicbrainz-dockerfile/Dockerfile

```diff
@@ -28,9 +27,8 @@ RUN curl -sSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
         python-minimal \
         zlib1g-dev
 
-RUN git clone https://github.com/metabrainz/musicbrainz-server.git musicbrainz-server && \
-    cd musicbrainz-server && \
-    git checkout v-2020-04-13
+ARG MUSICBRAINZ_VERSION=v-2020-04-13
+RUN git clone --depth=1 https://github.com/metabrainz/musicbrainz-server.git -c advice.detachedHead=false -b $MUSICBRAINZ_VERSION musicbrainz-server
```

Comment:
Same change as https://github.com/metabrainz/musicbrainz-docker/commit/28fd525454582d75287a08abae7c6852e0eeeef7 available in the development branch [`mbvm-38-dev`](https://github.com/metabrainz/musicbrainz-docker/tree/mbvm-38-dev) already.

Reply:
Good to know, thank you!